translations/ui-circulation/en.json

```diff
@@ -304,6 +304,8 @@
   "settings.titleLevelRequests.expirationNotice": "Expiration notice",
   "settings.titleLevelRequests.desription": "Select templates for the following patron notices. If no template is selected, no notice will be sent.",
   "settings.titleLevelRequests.defaultTemplateValue": "Select notice",
+  "settings.titleLevelRequests.forbiddenDisableTlrModal.title": "Cannot change \"Allow title level requests\"",
+  "settings.titleLevelRequests.forbiddenDisableTlrModal.description": "\"Allow title requests\" cannot be changed because it is in use by one or more requests",
```

Comment:
"Allow title level" => "Allow title-level"
"Allow title requests" => "Allow title-level requests"

Reply:
Thank you! Unfortunately my PO is on vacation for the rest of the week. But I promise to discuss it with her when she returns. I must resolve this conversation now, because other way this task will be a spillover for our team. Hope you have no any objections. Sorry for the rush and thank you again.